libs/sdk-backend-spi/src/backend/capabilities.ts

```diff
@@ -67,6 +67,11 @@ export interface IBackendCapabilities {
      */
     supportsElementsQueryParentFiltering?: boolean;
 
+    /**
+     * Indicates whether backend supports security settings.
+     */
+    supportsSecuritySettings?: boolean;
+
```

Comment:
i think we don't need this. the idea behind these capabilities is that they are something of a 'top level feature toggles' which can be used in app to disable some features because the backend will be throwing NotSupported for them anyway.

with the security settings, any backend can support them in a simple way: return true :)

Reply:
I removed it. Tiger backend have its own implementation now. It returns hardcoded value `false` to not allow to process dangerous control events that require whitelist validation.